spec/openapi.yaml

```diff
@@ -1481,7 +1481,7 @@ paths:
         
         ## The most basic request has the following structure:
           - ```sender_address``` is case insensitive. When one already exists for this account, you get a error message ```conflict```;
-          - ```destination_countries``` currently we only support ```AU``` and ```GB``` as the destination country;
+          - ```destination_countries``` currently we only support ```AU```, ```AT```, ```HR```, ```CY```, ```EE```, ```FI```, ```DE```, ```IE```, ```IT```, ```LV```, ```LT```, ```LU```, ```MT```, ```NL```, ```PT```, ```SI```, ```ES```, ```CH```, ```SE```, ```DK```, ```NO``` and ```GB``` as the destination country;
```

Comment:
I wonder if it would be better to keep this list as alphabetical?

Reply:
I had updated. Please help review and Merge it!

Thank you! @paultirta

Reply:
Still one incorrect. **CH** shouldn't be the last one in the list.

Reply:
Done!